src/Culling/babylon.boundingSphere.ts

```diff
@@ -44,8 +44,9 @@
          * Recreates the entire bounding sphere from scratch
          * @param min defines the new minimum vector (in local space)
          * @param max defines the new maximum vector (in local space) 
+         * @param worldMatrix defines the new world matrix
          */
-        public reConstruct(min: Vector3, max: Vector3) {
+        public reConstruct(min: Vector3, max: Vector3, worldMatrix?: Matrix) {

```

Comment:
adding an optional worldMatrix in case we want to update both local and world bounds